solidity/test/TestPhasedEscrow.js

```diff
@@ -249,6 +268,157 @@ describe("PhasedEscrow", () => {
         })
       })
     })
+
+    describe("when withdrawing to a staker rewards escrow beneficiary", () => {
+      const baseBalance = 200000000
+      let stakingContract
+      let operatorContract
+      let rewardsBeneficiary
+      let phasedEscrow
+
+      before(async () => {
+        const contracts = await initContracts(
+          contract.fromArtifact("TokenStaking"),
+          contract.fromArtifact("KeepRandomBeaconService"),
+          contract.fromArtifact("KeepRandomBeaconServiceImplV1"),
+          contract.fromArtifact("KeepRandomBeaconOperatorBeaconRewardsStub")
+        )
+
+        stakingContract = contracts.stakingContract
+        operatorContract = contracts.operatorContract
+
+        phasedEscrow = await PhasedEscrow.new(token.address, {from: owner})
+        const amount = web3.utils.toBN(baseBalance)
+        await token.transfer(phasedEscrow.address, amount, {from: owner})
+      })
+
+      beforeEach(createSnapshot)
+      afterEach(restoreSnapshot)
```

Comment:
We don't need them, `beforeEach` and `afterEach` from lines 54-58 (under the main `describe` still work here).

Reply:
Removed.